src/app/analyze/stack/stack-details/stack-details.component.ts

```diff
@@ -188,10 +188,10 @@ export class StackDetailsComponent implements OnInit {
     const url: string = this.similarStacks[0].uri;
     this.recoArray[this.currentIndex]['rows'] = [];
     this.recoArray[this.currentIndex]['url'] = url;
-    for (var component in components) {
+    for (let component of components) {
```

Comment:
I was not sure about this one, you may want to use for .. in. It depends on what "components" is and what you getting out of it.